Assets/Tests/KotlinTokenizer/overrideargs.swift

```diff
@@ -0,0 +1,5 @@
+class Test {
+    override func dostuff(x:Int = 5) {
```

Comment:
Very minor, but it is good practice to try to keep formatting as close as possibe to the standard accepted Swift. In this case, the space between `x` and `Int`. Not a reason to reject the PR, but just a comment (I changed it myself)

Reply:
Ah, yes. I started using swift without spaces between label and type in arguments, but switched a while back, but still forget sometimes...